app/index-tmpl.html

```diff
@@ -91,13 +91,12 @@
             /*}*/
         </style>
 
-
+        <!-- Piwik -->
```

Comment:
Happy to move the piwik code into `<head>`. Can we also move the google analytics code too, to keep them together? 

We need to investigate further the consequenced of removing the `<noscript>` section, particularly around adblock.

Reply:
sure, I can move GA up.